app/static/app/js/components/ProjectListItem.jsx

```diff
@@ -265,6 +266,8 @@ class ProjectListItem extends React.Component {
             <span>Updating task information... <i className="fa fa-refresh fa-spin fa-fw"></i></span>
           : ""}
 
+          {this.state.showTaskList ? <TaskList ref={this.setRef("taskList")} source={`/api/projects/${this.props.data.id}/tasks/?ordering=-id`}/> : ""}
```

Comment:
Interesting use of `ref`s. This is inverse of how I'm used to using components, so I've been thinking through the implications. It's more explicit than flowing `props` down to children, though it does lose the `propTypes` validation that would occur when passing functions _to_ children (potentially triggering actions).

I've been happy with Redux and associated patterns in the past, but for my most recent React app I didn't bother with the wiring and am somewhat regretting the externalization of state.

(Irrelevant aside done ;-)